### PR TITLE
Changes to move telegraf out of /etc/opt

### PR DESCRIPTION
**Pull request by linsomniac**
Issue #264: This attempts to fix the /etc/opt and put the binary
linked into /bin

Note: I can't seem to find the right combination of packages for
getting package.sh to run, so I haven't been able to test this.

### CODE REVIEW

File: scripts/init.sh [hunk 1]
```diff
@@ -95,7 +95,7 @@ function log_success_msg() {
 name=telegraf
 
 # Daemon name, where is the actual executable
-daemon=/opt/telegraf/telegraf
+daemon=/bin/telegraf
```
Comment (ekini) [inline scripts/init.sh]: /usr/bin/telegraf, then?

Comment (sparrc) [inline scripts/init.sh]: yes, this should be /usr/bin

Comment (linsomniac) [inline scripts/init.sh]: Oops, fixed.

**Comment by sparrc** [comment]
@linsomniac please also edit the README with the new paths, thank you!

**Comment by linsomniac** [comment]
Ahh, good point.  Thanks, pushed changes.

File: README.md [hunk 2]
```diff
@@ -36,8 +36,8 @@ Latest:
 
 ##### Package instructions:
 
-* Telegraf binary is installed in `/opt/telegraf/telegraf`
-* Telegraf daemon configuration file is in `/etc/opt/telegraf/telegraf.conf`
+* Telegraf binary is installed in `/usr/sbin/telegraf`
```
Comment (sparrc) [inline README.md]: `/usr/bin`

Comment (linsomniac) [inline README.md]: Argh, thanks!  Sorry, distracted this afternoon. :-/

**Comment by ekini** [comment]
You also have to change paths in `scripts/telegraf.service`.

**Comment by linsomniac** [comment]
Done.

**Comment by sparrc** [comment]
cc @rossmcdonald FHS changes for Telegraf

**Comment by padcom** [comment]
@linsomniac Is there a chance to update the branch so that it is in line with the latest?

**Comment by sparrc** [comment]
closing this in favor of #497

**Pull request closed without merging**